src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -397,22 +302,7 @@ func (cfg *Config) applyClientDefaults() {
 	if cfg.Client.FrontedServers == nil {
 		cfg.Client.FrontedServers = make([]*client.FrontedServerInfo, 0)
 	}
-	if len(cfg.Client.FrontedServers) == 0 && len(cfg.Client.ChainedServers) == 0 {
-		/*
-			cfg.Client.FrontedServers = []*client.FrontedServerInfo{
-				&client.FrontedServerInfo{
-					Host:           defaultRoundRobin(),
-					Port:           443,
-					PoolSize:       0,
-					MasqueradeSet:  cloudflare,
-					MaxMasquerades: 20,
-					QOS:            10,
-					Weight:         4000,
-					Trusted:        true,
-				},
-			}
-
-		*/
```

Comment:
If we're not using FrontedServers anymore, perhaps we should remove them from the client config.